File: dotnet/cafdemalihapa/Directories/Hosting/Feature/Calls/CallsDirectory.cs
namespace cafdemalihapa.Directories.Hosting.Feature.Calls
{
    public class CallsDirectory()
    {
        public string GetPath()
        {
            var directoryName = "calls";
            var featureDirectoryPath = FeatureDirectory.GetPath();
            var notesAndMessages = Path.Combine(featureDirectoryPath, directoryName);
            return notesAndMessages;
        }
        public void Create()
        {
            var path = GetPath();
            if(!Directory.Exists(path))
            {
                Directory.CreateDirectory(path);
            }
        }
    }
}

File: dotnet/cafdemvers/zzq-cafdemwimu/cafdemwimu.console/Directories/Hosting/Feature/Calls/CallsDirectory.fs
namespace cafdemwimu.console.Directories.Hosting.Feature.Calls

open System.IO
open cafdemwimu.console.Directories.Hosting.Feature

type CallsDirectory() =
    member _.GetPath() =
        let directoryName = "calls"
        let featureDirectoryPath = FeatureDirectory.GetPath()
        let notesAndMessages = Path.Combine(featureDirectoryPath, directoryName)
        notesAndMessages

    member this.Create() =
        let path = this.GetPath()
        if not (Directory.Exists(path)) then
            Directory.CreateDirectory(path) |> ignore
